### PR TITLE
Wallet functions

### PR DESCRIPTION
**Pull request by unconst**
Adds the following properties to the wallet:

wallet.uid
-- returns the hotkey uid
wallet.stake
-- returns the hotkey stake
wallet.balance
--returns the coldkey balance

Adds the following functions to the wallet:

wallet.add_stake()
-- adds stake to the hotkey.
wallet.remove_stake()
--removes stake from the hotkey.

### CODE REVIEW

File: bittensor/_subtensor/subtensor_impl.py [hunk 1]
```diff
@@ -285,56 +227,63 @@ def subscribe(
...
+            if wait_for_inclusion or wait_for_finalization:
+                response.process_events()
+                if response.is_success:
+                    bittensor.logging.success( 'Staked', str(amount.tao) )
+                    return True
+                else:
+                    bittensor.logging.warning( 'Failed to stake', str(response.error_message) )
+                    return False
+            else:
+                return True
```
Comment (shibshib) [inline bittensor/_subtensor/subtensor_impl.py:286]: Don't think we need an else here, can just return True. Same for the else on line 282.

File: bittensor/_subtensor/subtensor_impl.py [hunk 2]
```diff
@@ -377,13 +323,22 @@ def transfer(
...
+            if wait_for_inclusion or wait_for_finalization:
+                response.process_events()
+                if response.is_success:
+                    bittensor.logging.success( 'Transfered', str(amount) + '->' + str(dest) )
+                    return True
+                else:
+                    bittensor.logging.warning( 'Failed to transfer:', str(response.error_message) )
+                    return False
+            else:
+                return True
```
Comment (shibshib) [inline bittensor/_subtensor/subtensor_impl.py:336]: Same comment as above.

File: bittensor/_subtensor/subtensor_impl.py [hunk 3]
```diff
@@ -394,29 +349,39 @@ def unstake(
...
+            if wait_for_inclusion or wait_for_finalization:
+                response.process_events()
+                if response.is_success:
+                    bittensor.logging.success( 'Unstaked', str(amount.tao) )
+                    return True
+                else:
+                    bittensor.logging.warning( 'Failed to unstake:', str(response.error_message) )
+                    return False
+            else:
+                return True
```
Comment (shibshib) [inline bittensor/_subtensor/subtensor_impl.py:384]: Same thing.

**Review by shibshib — APPROVED** [review]

File: bittensor/_subtensor/subtensor_impl.py (hunk 1 above)
Comment (unconst) [reply bittensor/_subtensor/subtensor_impl.py:286]: mega nit

**Pull request merged**